Adyen/Integration/01_ManageAdyenSettings.md

```diff
@@ -0,0 +1,282 @@
+
+
+#### Procedure
+
+*Adyen&trade; Backend > Developers > Webhooks*
+
+![Webhook configuration in Adyen](../../Assets/Screenshots/Adyen/Integration/AdyenWebhooks.png)
+
+1. Click the [+ Webhook] button and select the [Add] button at the Standard webhook entry.   
+    The settings for the standard webhook are displayed.
+
+   ![Create webhook in Adyen](../../Assets/Screenshots/Adyen/Integration/AdyenCreateWebhook.png "[Create webhook in Adyen]")
+
+2. Enter the following settings in the *General* section.
+    - *Description*  
+        Enter a description for the webhook.
+    - *Server configuration section*   
+         - *URL*   
+           Enter the server URL of the Actindo productive system or sandbox you want to connect.    
+           For example: `https://customeraccount.dev.actindo.com/Actindo.Extensions.Actindo.Adyen.Notification.notificationAsync`. This example displays the server URL for asynchronous processing. For detailed information on synchronous processing, see [Define synchronous or asynchronous processing](#define-synchronous-or-asynchronous-processing). 
+      
+         - *Method*   
+            You can use the standard setting **JSON** if you have no other requirements.
+
+        - *Encryption protocol*     
+           You can use the standard setting **TLSv1.3** if you have no other requirements
+
+        - *Service version*   
+            Displays the version number.
+            
+          ![Define server configuration](../../Assets/Screenshots/Adyen/Integration/AdyenCreateWebhookServerConfig.png "[Define server configuration]").
+
+3. Click the [Apply] button and proceed with the following fields in the *General* section.
+
+    - *Merchant accounts*   
+        Enter the merchant account(s) you want to connect.
+    - *Events*   
+        Select all the events whose messages you want to transfer to the *Payments* module. All listed events can be handled by Actindo. You can exclude an event if you want to process specific events by another third-party application, for example. Alternatively, you can define the events to be ignored in the *Payments* module settings, see [Configure Adyen connection](./02_ManageAdyenConnection.md#configure-adyen-connection).
+
+        ![Events](../../Assets/Screenshots/Adyen/Integration/AdyenEvents.png "[Events]")
+
+3. Click the [Apply] button and enter the following settings in the *Security* section.
+
+     - *Basic authentication*   
+         - Define a username and password for the notification user, so that you can get notifications from Adyen&trade;. You can freely define them.
+
+            ![Basic authentification](../../Assets/Screenshots/Adyen/Integration/BasicAuthentification.png "[Basic authentification]")
+
+         - Change to your Actindo instance and enter the server's username and password in the *Notification user* and *Notification password* fields the *Payments* settings. 
```

Comment:
"server's username and password" -> Oben haben wir "server" gar nicht erwähnt; wir reden über "Basic authentication".  Ich gehen davon aus, die Details, die man hier angeben soll, sind diejenigen, die wir soeben in Adyen angegeben haben. Ich fand aber dieses "server" verwirrend.
 Vorschlag: "... enter the username and password (previously) defined in the *Basic authenticaton* section" (oder so was).